### PR TITLE
Fix Py3 linter script.

### PR DESCRIPTION
**Pull request by tvalentyn**
[BEAM-1251] Make futurize-based checker in py27-lint3 suite running again.
------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

It will help us expedite review of your Pull Request if you tag someone (e.g. `@username`) to look at it.

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_GradleBuild/lastCompletedBuild/) | --- | --- | --- | --- | --- | ---
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_GradleBuild/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark_Gradle/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) </br> [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_VR_Flink/lastCompletedBuild/) | --- | --- | ---

### CODE REVIEW

**Comment by tvalentyn** [comment]
R: @RobbeSneyders @aaltay

File: sdks/python/scripts/run_pylint_2to3.sh [hunk 1]
```diff
@@ -47,17 +47,23 @@ if test $# -gt 0; then
   esac
 fi
 
-FUTURIZE_GREP_PARAM=$( IFS='|'; echo "${ids[*]}" )
+EXCLUDED_FROM_FUTURIZE_CHECK=(
+# Non-portable code runs only in Python 2.
+# https://github.com/apache/beam/blob/7b8786f84739f4f50c2491cb247b75459734b5ac/sdks/python/apache_beam/typehints/typehints.py#L343
+"apache_beam/typehints/typehints.py"
```
Comment (RobbeSneyders) [inline sdks/python/scripts/run_pylint_2to3.sh]: Could we just locally ignore the thrown pylint warning here, so we don't have to exclude the full module?

Comment (tvalentyn) [reply sdks/python/scripts/run_pylint_2to3.sh]: It's not a pylint warning, but an output from futurize. Futurize produces the following output
```
RefactoringTool: Refactored apache_beam/typehints/typehints.py
--- apache_beam/typehints/typehints.py  (original)
+++ apache_beam/typehints/typehints.py  (refactored)
@@ -341,7 +341,7 @@
   # Must either be a TypeConstraint instance or a basic Python type.
   possible_classes = [type, TypeConstraint]
   if sys.version_info[0] == 2:
-    possible_classes.append(types.ClassType)
+    possible_classes.append(type)
   is_not_type_constraint = (
       not isinstance(type_param, tuple(possible_classes))
       and type_param is not None)
```
From which the script filters out the line ```RefactoringTool: Refactored apache_beam/typehints/typehints.py```, which does not have much details except for filename. We could add a directive in the code for ```futurize``` to ignore a line, but I did not see a way to do it - do we have one?

**Comment by tvalentyn** [comment]
PTAL

**Comment by RobbeSneyders** [comment]
LGTM.
I think the Python PreCommit should pass if you rebase on top of master, since this was already fixed in #6489.

**Comment by tvalentyn** [comment]
retest this please

**Comment by tvalentyn** [comment]
Thanks!

**Pull request merged**